WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -313,13 +309,13 @@ private ImmutableList<SmartCoin> SelectCoinsForRound(IEnumerable<SmartCoin> coin
 			.ToArray();
 
 		// How many inputs do we want to provide to the mix?
-		int inputCount = ConsolidationMode ? MaxInputsRegistrableByWallet : GetInputTarget(filteredCoins.Length);
+		int inputCount = GetInputTarget(filteredCoins.Length);
```

Comment:
ConsolidationMode guaranteed that we will use all of the coins. We can refactor it but now it is just removed without replacing it with something else.

Reply:
It is not used, except in tests. We shouldn't have completely different behaviors for tests and real production. This method is already too messy and buggy. Let remove the unused flag and once we have something working we can add it back if needed.

Reply:
Ahhh it was added to make the test pass. It doesn't look right.

Reply:
It smells like a hack I agree, but it is working and makes sense. There could be something more elegant like a mocked SelectCoinsForRound that is selecting all the coins no matter what. I would not care about it now.

Reply:
The problem is that the tests are there to let us know when something is wrong and in this case the tests fail so, how do we know this will not fail in production too?

Reply:
I think we have to write tests that is testing that feature individually.